tools/docker/README.md

```diff
@@ -4,7 +4,7 @@ This folder contains scripts and Dockerfiles for users to build the Docker image
 for Occlum. An Occlum Docker image sets up the development environment for
 Occlum and also gets Occlum preinstalled.
 
-Currently, three Linux OS distributions are supported: Ubuntu 18.04, CentOS 8.2 and aliyunlinux3.
+Currently, three Linux OS distributions are supported: Ubuntu 18.04, CentOS 8.2, aliyunlinux3 and anolis8.4.
```

Comment:
This doc is a little out of data. Can you please help to add "ubuntu20.04" in the list?

Reply:
> This doc is a little out of data. Can you please help to add "ubuntu20.04" in the list?

Sure, "ubuntu20.04" has been added in the list in the new commit.